tests/test_deployment.py

```diff
@@ -0,0 +1,572 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+import mock
+import os
+import pexpect
+import pytest
+import six
+import subprocess
+import sys
+import tempfile
+from pytest import raises
+from six.moves import configparser
+
+from dallinger.deployment import new_webbrowser_profile
+from dallinger.command_line import verify_package
+from dallinger.config import get_config
+from dallinger import recruiters
+
+
+def found_in(name, path):
+    return os.path.exists(os.path.join(path, name))
+
+
+@pytest.fixture
+def output():
+
+    class Output(object):
+
+        def __init__(self):
+            self.log = mock.Mock()
+            self.error = mock.Mock()
+            self.blather = mock.Mock()
+
+    return Output()
```

Comment:
why not just `return mock.Mock()`?

Reply:
Because log, error, and blather are attributes rather than methods, `Mock` determine its API with the `spec` argument (`mock.Mock(spec=dallinger.command_line.Output)`).  I generally try to do this to provide some protection against mocks getting out of sync with what they're mocking.